lib/vmpooler/pool_manager.rb

```diff
@@ -189,7 +189,7 @@ def clone_vm(template, folder, datastore, target)
 
         # Generate a randomized hostname
         o = [('a'..'z'), ('0'..'9')].map(&:to_a).flatten
-        vm['hostname'] = o[rand(25)] + (0...14).map { o[rand(o.length)] }.join
+        vm['hostname'] = $config[:config]['prefix'] + o[rand(25)] + (0...14).map { o[rand(o.length)] }.join
```

Comment:
Is the separator between `prefix` and random name expected to be contained within the `prefix` itself?